sdk/maps/azure-maps-search/azure/maps/search/_search_client.py

```diff
@@ -65,1050 +45,387 @@ def __init__(
+         borders and labels to align with the specified user region. By default, the View parameter is
+         set to “Auto” even if you haven’t defined it in the request.
+
+         Please refer to `Supported Views <https://aka.ms/AzureMapsLocalizationViews>`_ for details and
+         to see the available Views. Default value is None.
+        :paramtype view: str
+        :keyword coordinates: A point on the earth specified as a longitude and latitude. When you
+         specify this parameter, the user’s location is taken into account and the results returned may
+         be more relevant to the user. Example: &coordinates=lon,lat. Default value is None.
+        :paramtype coordinates: list[float]
+        :keyword admin_district: The country subdivision portion of an address, such as WA.
+
+         **If query is given, should not use this parameter.**. Default value is None.
+        :paramtype admin_district: str
+        :keyword admin_district2: The county for the structured address, such as King.
+
+         **If query is given, should not use this parameter.**. Default value is None.
+        :paramtype admin_district2: str
+        :keyword admin_district3: The named area for the structured address.
+
+         **If query is given, should not use this parameter.**. Default value is None.
+        :paramtype admin_district3: str
+        :keyword locality: The locality portion of an address, such as Seattle.
+
+         **If query is given, should not use this parameter.**. Default value is None.
+        :paramtype locality: str
+        :keyword postal_code: The postal code portion of an address.
+
+         **If query is given, should not use this parameter.**. Default value is None.
+        :paramtype postal_code: str
+        :return: GeocodingResponse
+        :rtype: ~azure.maps.search.models.GeocodingResponse
+        :raises ~azure.core.exceptions.HttpResponseError:
         """
-        batch_items = [{"query": f"?query={query}"} for query in search_queries] if search_queries else []
 
-        return self._search_client.begin_fuzzy_search_batch(
-            batch_request={"batch_items": batch_items},
+        result = self._search_client.get_geocoding(
+            top=top,
+            query=query,
+            address_line=address_line,
+            country_region=country_region,
+            bbox=bbox,
+            view=view,
+            coordinates=coordinates,
+            admin_district=admin_district,
+            admin_district2=admin_district2,
+            admin_district3=admin_district3,
+            locality=locality,
+            postal_code=postal_code,
             **kwargs,
         )
-
+        return result
 
     @distributed_trace
-    def search_address_batch(
+    def get_geocoding_batch(
         self,
-        search_queries: List[str],
+        geocoding_batch_request_body: Union[_models.GeocodingBatchRequestBody, IO[bytes]],
```

Comment:
We should avoid user to use `IO[bytes]`, it's not user friendly format.

Reply:
Actually, all of our APIs should support `IO[bytes]` to pass through a request body. However I don't think we need to wrap these methods, we can just expose the generated ones directly.